liquidaty/zsv
# zsv playground

- [x] Separate job for wasm build
- [x] Separate job for deploying wasm build to GitHub Pages
- [x] Fix compiler warnings
- [x] Fix test coloring issue (replace `echo -e` with `printf`)
- [x] Ignore generated files
- [ ] <strike>Add support for `stdin`</strike>
  - not feasible to handle generically both for csv input and command line prompting
- [x] Add support for adding files
  - With multiple browsers e.g. chrome, firefox, etc.
  - Make sure the cache is disabled during testing to avoid any surprises and/or side effects later

Other misc updates:

- Use `NO_THREADING` to skip `-pthread` flag
- Use `NO_STDIN` to disable stdin input for consistency
- Introduce `NO_PLAYGROUND` CLI flag for conditional switching of flags and main function name for playground and the rest

Resolves #161

TODOs:

- Verify deployment after merge to `main`
- Fix issues if any

Signed-off-by: Azeem Sajid <azeem.sajid@gmail.com>
